org.kde.ghostwriter.yml

```diff
@@ -0,0 +1,66 @@
+app-id: org.kde.ghostwriter
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: ghostwriter
+rename-icon: ghostwriter
+finish-args:
+- "--socket=wayland"
+- "--socket=x11"
+- "--socket=pulseaudio"
+- "--share=ipc"
+- "--share=network"
+- "--filesystem=home"
+- "--device=dri"
+- "--env=QTWEBENGINE_DISABLE_SANDBOX=1"
+modules:
+- name: cmark
+  buildsystem: cmake-ninja
+  builddir: true
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=Release"
+  sources:
+  - type: git
```

Comment:
You should use tarballs as much as possible

Reply:
Couldn't find any release tarball ...